### ISSUE TITLE
After pressing 'Edit' button on Finish and review page, Screen reader announces 'Next' only while focus is on the page.

### ISSUE THREAD

**Issue by msftedad** [nvaccess/nvda#13202]
### Steps to reproduce:
Step 1: Open the URL in Dev browser.
Step 2: Login with valid credentials.
Step 3: Admin Portal Home page will be displayed
Step 4: Navigate the Users TAB and activate 'Add user' menu item.
Step 5: Navigate to optional settings page. 
Step 6: Navigate to 'Finish and review' page.
Step 7: Select 'Edit' button and observe the issue.

### Actual behavior:
Screen Reader announces "Next" button information only after selecting 'Edit' button on finish page.

### Expected behavior:
When focus lands on the page, screen Reader should announce the page heading information with 'Next' button. 

### Additional Notes:
This issue occurs on each edit button available on finish and review page.

### System configuration
Application: M365 Admin Portal
URL: https://admin.microsoft.com/Adminportal/Home
Browser: Edge, Version 96.0.1054.62 (Official build) (64-bit)

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
NVDA Version 2021.1

#### Windows version:
Window 11, Version 21H2 (OS Build 22000.376)

#### Does the issue still occur after restarting your computer?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Yes, also reproducible through NVDA version 2020.4

**Comment by feerrenrut** [comment]
This needs further investigation by Microsoft developers. Please provide a minimal test case for us to investigate. I'm closing this issue for now.
